app/assets/stylesheets/site.scss

```diff
@@ -251,21 +251,6 @@ tbody tr:hover td {
   text-decoration: none;
 }
 
-.usa-table {
```

Comment:
These overrides were replaced by setting `$theme-table-border-color` in `uswds/_uswds-theme.scss`.